app/assets/stylesheets/stream_element.scss

```diff
@@ -63,10 +63,19 @@
 #main_stream .stream_element {
   padding: 10px;
```

Comment:
You also added `padding: 10px;` to the `.stream_element` rule so I guess you can remove this here.